cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -476,18 +479,43 @@ func main() {
 			}),
 		)
 
+		// We use the V1 CSIStorageCapacity API if available.
+		clientFactory := capacity.NewV1ClientFactory(clientset)
+		cInformer := factoryForNamespace.Storage().V1().CSIStorageCapacities()
+
+		invalidCapacity := &storagev1.CSIStorageCapacity{
```

Comment:
Can you add some comments before line 486 explaining why you are trying to create an object with an invalid name?  Although people will get it when reading the code below, it is better to clarify earlier.

Reply:
Done.